examples/ginwebapp/main.go

```diff
@@ -52,87 +47,30 @@ func main() {
 		os.Exit(1)
 	}
 
-	r.GET("/otp/signup", handleSignUp)
-	r.GET("/otp/signin", handleSignIn)
+	r.GET("/", help)
+	r.GET("/otp/signupOrIn", handleSignUpOrIn)
 	r.GET("/otp/verify", handleOTPVerify)
 
-	r.GET("/oauth", handleOAuth)
-
-	r.GET("/magiclink/signin", handleMagicLinkSignIn)
-	r.GET("/magiclink/signup", handleMagicLinkSignUp)
-	r.GET("/magiclink/session", handleGetMagicLinkSession)
-	r.GET("/magiclink/verify", handleMagicLinkVerify)
-
-	r.GET("/webauthn", func(c *gin.Context) {
-		file, _ := os.ReadFile("../demo.html")
-		c.Data(http.StatusOK, "text/html; charset=utf-8", file)
-	})
-
-	r.POST("/webauthn/signup/start", func(c *gin.Context) {
-		decoder := json.NewDecoder(c.Request.Body)
-		var t *auth.User
-		err := decoder.Decode(&t)
-		if err != nil {
-			setError(c, err.Error())
-			return
-		}
-
-		res, err := client.Auth.WebAuthn().SignUpStart(t.Name, t, c.Query("origin"))
-		if err != nil {
-			setError(c, err.Error())
-		}
-		c.Writer.Header().Add("Content-Type", "application/json")
-		c.PureJSON(http.StatusOK, res)
-	})
-	r.POST("/webauthn/signup/finish", func(c *gin.Context) {
-		decoder := json.NewDecoder(c.Request.Body)
-		var t *auth.WebAuthnFinishRequest
-		err := decoder.Decode(&t)
-		if err != nil {
-			setError(c, err.Error())
-			return
-		}
-
-		_, err = client.Auth.WebAuthn().SignUpFinish(t, c.Writer)
-		if err != nil {
-			setError(c, err.Error())
-		}
-		setOK(c)
-	})
-
-	r.POST("/webauthn/signin/start", func(c *gin.Context) {
-		res, err := client.Auth.WebAuthn().SignInStart(c.Query("id"), c.Query("origin"), nil, nil)
-		if err != nil {
-			setError(c, err.Error())
-		}
-		c.Writer.Header().Add("Content-Type", "application/json")
-		c.PureJSON(http.StatusOK, res)
-	})
-	r.POST("/webauthn/signin/finish", func(c *gin.Context) {
-		decoder := json.NewDecoder(c.Request.Body)
-		var t *auth.WebAuthnFinishRequest
-		err := decoder.Decode(&t)
-		if err != nil {
-			setError(c, err.Error())
-			return
-		}
-
-		_, err = client.Auth.WebAuthn().SignInFinish(t, c.Writer)
-		if err != nil {
-			setError(c, err.Error())
-		}
-		setOK(c)
-	})
-
 	authorized := r.Group("/")
 	authorized.Use(descopegin.AuthneticationMiddleware(client.Auth, nil, nil))
 	authorized.GET("/private", handleIsHealthy)
-	authorized.POST("/logout", handleLogout) // Logout from all user's active sessions
+	authorized.GET("/logout", handleLogout) // Logout from all user's active sessions
 	r.RunTLS(fmt.Sprintf(":%s", port), TLSCertPath, TLSkeyPath)
 }
 
+func help(c *gin.Context) {
+	helpTxt := "To test sign up or in with otp email go to /otp/signupOrIn?email=\n\n"
```

Comment:
- Let's add `:` after the go to in each line so it will be clearer.
- I don't think we need the "To" at the beginning of every line, we can have an header above all options.